src/pudl/helpers.py

```diff
@@ -287,45 +304,44 @@ def oob_to_nan(df, cols, lb=None, ub=None):
 
 def oob_to_nan_with_dependent_cols(
     df: pd.DataFrame,
-    cols: list,
-    dependent_cols: list,
-    lb: float = None,
-    ub: float = None,
-):
-    """Call oob_to_nan and additionally nullify any derived columns.
+    cols: list[str],
+    dependent_cols: list[str],
+    lb: float | None = None,
+    ub: float | None = None,
+) -> pd.DataFrame:
+    """Call :func:`oob_to_nan` and additionally nullify any derived columns.
 
     Set values in ``cols`` to NaN if values are non-numeric or outside of a
     given range. The corresponding values in ``dependent_cols`` are then set
     to NaN. ``dependent_cols`` should be columns derived from one or multiple
     of the columns in ``cols``.
 
     Args:
-        df (pandas.DataFrame): The dataframe containing values to be altered.
-        cols (iterable): Labels of the columns whose values are to be changed.
-        dependent_cols (iterable): Labels of the columns whose corresponding
-            values should also be nullified. Columns are derived from one or
-            multiple of the columns in ``cols``.
-        lb: (number): Lower bound, below which values are set to NaN. If None,
-            don't use a lower bound.
-        ub: (number): Upper bound, below which values are set to NaN. If None,
-            don't use an upper bound.
+        df: The dataframe containing values to be altered.
+        cols: Labels of the columns whose values are to be changed.
+        dependent_cols: Labels of the columns whose corresponding values should also be
+            nullified. Columns are derived from one or multiple of the columns in
+            ``cols``.
+        lb: Lower bound, below which values are set to NaN. If None, don't use a lower
+            bound.
+        ub: Upper bound, below which values are set to NaN. If None, don't use an upper
+            bound.
 
     Returns:
-        pandas.DataFrame: The altered DataFrame.
+        The altered DataFrame.
     """
     out_df = oob_to_nan(df, cols, lb, ub)
     out_df.loc[out_df[cols].isnull().any(axis=1), dependent_cols] = np.nan
     return out_df
 
 
-def prep_dir(dir_path, clobber=False):
+def prep_dir(dir_path: str | pathlib.Path, clobber: bool = False) -> pathlib.Path:
     """Create (or delete and recreate) a directory.
 
     Args:
-        dir_path (path-like): path to the directory that you are trying to
-            clean and prepare.
-        clobber (bool): If True and dir_path exists, it will be removed and
-            replaced with a new, empty directory.
+        dir_path: path to the directory that you are trying to clean and prepare.
+        clobber: If True and dir_path exists, it will be removed and replaced with a
+            new, empty directory.
 
     Raises:
         FileExistsError: if a file or directory already exists at dir_path.
```

Comment:
For consistency, scratch the pathlib.Path type in the "returns" portion of the description as well.

Reply:
Oops! Missed this one. Updated.